src/simple_openid_connect/data.py

```diff
@@ -200,6 +201,9 @@ class Config:
     sid: Optional[str]
     "OPTIONAL. Session ID - String identifier for a Session. This represents a Session of a User Agent or device for a logged-in End-User at an RP. Different sid values are used to identify distinct sessions at an OP. The sid value need only be unique in the context of a particular issuer. Its contents are opaque to the RP."
 
+    raw_token: Optional[str]
+    "The raw token received from the issuer."
+
```

Comment:
I *really* don't like this since these data classes are supposed to be as close to the OpenId spec as possible. I don't have a better idea right now but I'll think about it :/